bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1437,7 +1436,7 @@ && approvalNeeded() && approval.approvalComment() && readyToPostApprovalNeededCo
             }
 
             // Calculate current metadata to avoid unnecessary future checks
-            var metadata = workItem.getMetadata(workItem.getPRMetadata(censusInstance, title, updatedBody, pr.comments(), activeReviews,
```

Comment:
This one is an actual bug. It could prevent a future run from realizing a relevant comment change took place since the last check run. It's important that the hash we save is done on the data we used when evaluating the PR. Thanks for fixing this!

Reply:
Yes, it's a bug and I didn't catch it when I was implementing [SKARA-1912](https://bugs.openjdk.org/browse/SKARA-1912).